formatTest/typeCheckedTests/expected_output/oo.re

```diff
@@ -30,28 +29,27 @@ let tmp = {
 /**
  * Comment on stackWithAttributes.
  */
```

Comment:
Just saw this comment. Yes I had to add special logic for the new behavior. I can revert it and it will also make the code simpler.